pkg/store/copr/coprocessor.go

```diff
@@ -1110,10 +1110,18 @@ func (it *copIterator) Next(ctx context.Context) (kv.ResultSubset, error) {
 	failpoint.InjectCall("CtxCancelBeforeReceive", ctx)
 	if it.liteWorker != nil {
 		resp = it.liteWorker.liteSendReq(ctx, it)
+		// after lite handle 1 task, reset tryCopLiteWorker to 0 to make future request can reuse copLiteWorker.
+		it.liteWorker.tryCopLiteWorker.Store(0)
```

Comment:
If this operation is **designed to be performed only once**, it is best to use CAS operations. This approach makes the code more robust, eliminating the need to consider the side-effects of additional settings being applied.

Reply:
done.